### PR TITLE
Disable metric export flag

### PR DESCRIPTION
**Pull request by aleksandra-malinowska**
Add flag to disable metrics export.

### CODE REVIEW

**Comment by piosz** [comment]
Looks good. Please rebase.

File: metrics/options/options.go [hunk 1]
```diff
@@ -82,4 +83,5 @@ func (h *HeapsterRunOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&h.HistoricalSource, "historical_source", "", "which source type to use for the historical API (should be exactly the same as one of the sink URIs), or empty to disable the historical API")
 	fs.BoolVar(&h.Version, "version", false, "print version info and exit")
 	fs.StringVar(&h.LabelSeperator, "label_seperator", ",", "seperator used for joining labels")
+	fs.BoolVar(&h.DisableMetricExport, "disable_export", false, "Disable exporting metrics in api/v1/")
```
Comment (piosz) [inline metrics/options/options.go]: api/v1/metric-export

File: metrics/api/v1/api_test.go [hunk 2]
```diff
@@ -28,7 +28,7 @@ import (
 
 func TestApiFactory(t *testing.T) {
```
Comment (piosz) [inline metrics/api/v1/api_test.go]: Please add a simple unit test.

**Review by piosz — CHANGES_REQUESTED** [review]

File: metrics/api/v1/api_test.go (hunk 2 above)
Comment (aleksandra-malinowska) [inline metrics/api/v1/api_test.go]: Done

**Comment by piosz** [comment]
LGTM

**Review by piosz — APPROVED** [review]

**Pull request merged**